site/site/cli/installation/npm.md

```diff
@@ -2,26 +2,28 @@
 
 The [default](./binary.md#default-build) build of Taplo is published to NPM as [`@taplo/cli`](https://www.npmjs.com/package/@taplo/cli). You will need [Node.js](https://nodejs.org/en/) (tested with 16 and up) in order to use it.
 
-## Installation via npm
```

Comment:
I would like to retain those headings

Reply:
+1 let's keep those headings

Reply:
Isn't this page already about installing with NPM though?

Reply:
Maybe the page title is out-of-date and should be renamed to Javascript or Node.js. I don't know this ecosystem, but it looks like npm, yarn, npx, and pnpm are all package managers for the Javascript runtime environment Node.js.

Reply:
The package is published to the NPM registry, but different tools can make installs faster than using the default `npm` CLI. I think the title "NPM" is fine but the second header probably isn't needed.

Reply:
I see, in that case the current state makes sense:
- The npm title means "NPM registry".
- The "Installation via npm" subtitle means "Installation via the npm tool".

Reply:
Exactly yeah. Can I resolve this?

Reply:
No, this is not too many headings. This is organization. Each heading creates a quick link on the top right such that it's easy to navigate to your favorite tool without scrolling and reading all the content.

Reply:
<img width="1063" alt="SCR-20230826-jznt" src="https://github.com/tamasfe/taplo/assets/47499684/c73ac64b-2e06-4b92-b299-d066a4c7cd38">
<img width="1078" alt="SCR-20230826-jzty" src="https://github.com/tamasfe/taplo/assets/47499684/c407ebbf-ee1c-40ee-b2b5-2d413daac6af">

I personally way prefer the first one. I can see all the content right away and I can easily find the command for my package manager of choice.

Reply:
I don't have a strong opinion and fine with both. @panekj what about you?

Reply:
I would like to keep the headings for organisational purpose (it divides the content and presents distinction in side menu)

Reply:
> I would like to keep the headings for organisational purpose (it divides the content and presents distinction in side menu)

Do the headings in the second image look good?